src/components/TagModal.vue

```diff
@@ -155,6 +177,34 @@ export default {
 				return `rgba(${r}, ${g}, ${b}, ${opacity})`
 			}
 		},
+		openEditTag() {
+			this.renameTagLabel = false
+			this.renameTagInput = true
+			this.showSaving = false
+
+		},
+		async renameTag(tag, event) {
+			this.renameTagInput = false
+			this.showSaving = false
```

Comment:
shouldn't this switch to `true` here …

Reply:
Technically yes, but that will add 1 second more to whole action to be saved. The saving its so fast that you cannot read the sting anyway. I think its unnecessary.

Reply:
doesn't saving depend on the connection to the server and how fast the server is able to respond?

Reply:
Yes, but showSaving - shows the popup that says "Saving..." not showing this popup saves 1-2 seconds. I dont have a strong opinion on this, i can show it if you think its helpful. We do the same on rename mailbox btw.